indicators/transferwise-d777126.yml

```diff
@@ -0,0 +1,20 @@
+title: Transferwise Phishing Kit d777126
+description: |
+    Transferwise phishing kit which uses the same sentry.io
+    API key across various domains.
+
+references:
+    - https://urlscan.io/result/6d6dc13f-671c-4c1b-a58c-cc9470bcef2d
+    - https://urlscan.io/result/c6a7e6a2-0b60-46ea-9741-ce53d816d729
+    - https://urlscan.io/result/86f8b7e2-1484-407c-a7b7-93a497014505
+    - https://urlscan.io/result/c2970a14-2b7a-449c-ade9-e4f6fa5c414d
+
+detection:
+
+    sentryAPICall:
+      requests|contains: '?sentry_key=d777126e0316438b8a582a674f5ec311&sentry_version=7'

```

Comment:
I suspect this is Wise's actual Sentry key but it does only seem to fire on phishing sits now 🤔

Reply:
Quite peculiar, I must say haha.

Reply:
It must be from the old version of Wise's login page, due to there being no reference to Sentry what so ever in the Wise login page now